test/suites/dev-tanssi-relay/collator-assignment/test-collator-assignment.ts

```diff
@@ -75,5 +75,11 @@ describeSuite({
                 expect(assignmentRandomness.isEmpty).toBe(false);
             },
         });
+
+        it({
+            id: "E02",
+            title: "Collator should rotate",
+            test: async function () {},
+        });
```

Comment:
Unfinished?

Reply:
*fixed* (I moved the test in separate file but forgot to remove it.)